invenio_rdm_records/mappings/v6/records/record-v1.0.0.json

```diff
@@ -22,20 +22,60 @@
           "type": "keyword"
         },
         "$schema": {
-          "type": "keyword"
+          "type": "text",
+          "index": false
         },
         "recid": {
-          "type": "long"
+          "type": "keyword"
         },
         "title": {
-          "type": "text"
+          "type": "text",
+          "copy_to": "suggest_title"
+        },
+        "suggest_title": {
+          "type": "completion"
         },
         "description": {
           "type": "text"
         },
         "owners": {
           "type": "integer"
         },
+        "keywords": {
+          "type": "keyword"
+        },
+        "publication_date": {
+          "type": "date",
+          "format": "date"
```

Comment:
normal: Why `date` here and `date-time` in the jsonschema above? (`date` doesn't include time)

Reply:
I'd swear I had answer this, somehow it didn't get through. Sorry.

The [Elasticsearch `date` type](https://www.elastic.co/guide/en/elasticsearch/reference/current/date.html) includes the time. There is actually no `date-time` in ES (AFAIK). But JSON Schema requires `date-time`. That's why the mapping has a different value than the jsonschema.